guides/V1_Meeting.md

```diff
@@ -0,0 +1,79 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'meeting'
+is: 'experimental'
+---
+
+# Get meeting details
+
+## Concepts
+
+The _meeting_ query allows the API caller to get details about the meeting.  The query accepts a Space ID (spaceId) and will return meeting details if the calling user has access to the meeting.  The calling user will have access to the meeting if the user has access to the space which contains the meeting, or the meeting has been marked to allow public participants.
+
+## Schema
+
+### Meeting Query
+
+
+```graphql
```

Comment:
The other guides use a tabular format. Most guides cover the documentation of a single type in the schema, though in the case of the moments doc, we used section headers per type with a table within each. The new fields on existing types, like the meeting field on QueryRoot, might be best demonstrated in a subset in the nav with an example?

Reply:
OK - I think that reactions guide isn't published into the nav yet @miguelestrada and most of the guides already in the nav use a more tabular format?

If we want something in this GraphQL type format - I think we should just _generate_ it in the GraphQL server itself. This is also not too different from what you actually already see in the sidebar of the GraphQL explorer, so not sure there is huge value in copying this here into a doc we need to update separately? I think the doc gives us the opportunity to say more about each field we add, and I'm not sure the graphql type language really allows us to do that.

I'm 100% for using these type definitions in our internal designs and specs, but I don't know if redundantly adding these into the doc when you can find something similar in the GraphQL explorer, is helpful.

One other thing we could explore in the future though - we could try to make the GraphQL explorer tool responsive to URLs that could open the doc sidebar to a specific type. Then we could make the type names clickable links perhaps and avoid redefining things here.